src/BMI/bmi_port.c

```diff
@@ -68,21 +67,56 @@ static inline int port_in_use(bmi_port_t *port) {
 }
 
 static inline bmi_port_t *get_port(bmi_port_mgr_t *port_mgr, int port_num) {
-  Word_t *pinfo;
-  JLG(pinfo, port_mgr->ports_map, (Word_t) port_num);
```

Comment:
`JLG` is also used in `src/bf_lpm_trie/bf_lpm_trie.c`.
Is it possible to remove the dependency there as well?